src/cloudfront/origin-access.ts

```diff
@@ -77,9 +77,6 @@ export const upsertOriginAccessControl = async (
     return existingOAC
   }
 
-  logger.info(
```

Comment:
Why remove this?

Reply:
it duplicates a similar log in the next function call